arch/s390/s390_features.c

```diff
@@ -1,10 +1,14 @@
 #include "../../zbuild.h"
 #include "s390_features.h"
 
-#include <sys/auxv.h>
+#if defined(HAVE_SYS_AUXV_H)
+#  include <sys/auxv.h>
+#endif
 
-Z_INTERNAL int s390_cpu_has_vx;
+Z_INTERNAL int s390_cpu_has_vx = 0;
```

Comment:
Aren't all static variables automatically initialized to zero?

Reply:
We explicitly declare them on other files too...

Reply:
Ok